app/controllers/banned_users_controller.rb

```diff
@@ -3,13 +3,13 @@ class BannedUsersController < ApplicationController
   before_action :set_username, only: [:create]
   before_action :is_admin?, only: [:create]
   def create  
-    redirect_to mod_path(@community) and return flash[:alert] = t(:not_present_to_ban, username: @username.username) unless @subscriptions.any?
+    redirect_to mod_path(@community) and return flash[:alert] = t("banned.user_not_present", username: @username.username) unless @subscriptions.any?
```

Comment:
banned.user_not_present is misleading, change it to banned.user_not_in_community

Reply:
changed and resolved